### ISSUE TITLE
mooclet backend: metric logs sync to rewards

### ISSUE THREAD

**Issue by danoswaltCL** [CarnegieLearningWeb/UpGrade#1934]

**Comment by zackcl** [comment]
Here is the example design of the read-only Mooclet Reward Metric section:

<img width="1000" alt="Image" src="https://github.com/user-attachments/assets/07cfc2a5-8b2b-418b-80b4-118f5219f60d" />

**Comment by danoswaltCL** [comment]
@zackcl looks good, I created a separate story for the stepper part so it can be worked in parallel #2203
